http-gateway/web/public/auth_config.json

```diff
@@ -1,6 +1,6 @@
 {
-  "domain": "ocfcloud.eu.auth0.com",
-  "clientId": "LXZ9OhKWWRYqf12W0B5OXduqt02q0zjS",
+  "domain": "auth.plgd.cloud",

```

Comment:
@jkralik please remove these default values. They shouldn't be in a container by defualt.